docs/interpreting-experiments/experiment-progress-bar.md

```diff
@@ -18,7 +20,29 @@ You can also see the progress bar on the details page for each experiment.
 
 **Note:** We compute the days remaining using a linear interpolation. This interpolation does not take into account that gathering data usually slows down during an experiment, and so the estimate may be optimistic, especially in the early days of an experiment.
 
-#### How we calculate the progress
+### How to use the progress bar
+
+Traditionally, when using a fixed sample test, you decide up front how long the experiment ought to run and cannot interpret results until you have finished gathering all data. However, the sequential testing approach we use allows for flexibility in deciding when to stop an experiment. Here's some advice on how to get the most out of the progress bar.
+
+Before starting with any experiment, we have to set the minimum detectable effect to something that makes sense for your use case. Two aspects are important:
+- What constitutes a meaningful impact to the business: a 0.1% increase in revenue can be valuable for a company that operates at the scale of Google, but for a start-up that looks to grow 3x year over year, this would make for a negligible impact. 
+- What effect sizes are you able to detect in a reasonable amount of time given the data volume. It's important to keep in mind that, as a rule of thumb, detecting an effect twice as small takes 4 times as long. Thus if based on your data volume it takes a month to detect a 10% lift, then it would take 4 months to detect a 5% lift. A constraint on the maximum time you are comfortable running an experiment often limits what MDE you can aim for.
+
+Keeping the above points in mind, we have set the MDE to a reasonable value, for illustrative purposes let's say at 5%, and we started our experiment. Now how do we use the progress bar? In two ways:
```

Comment:
Again, I would try to remain consistent in pronouns.

"Suppose, given the above considerations, you decide to set your MDE at 5%. You start your experiment, and the progress bar begins to fill up. How should you use it to make decisions?

1. The _color_ of the progress bar indicates whether or not the _primary_ metric is significant and positive. Blue indicates it is, gray indicates it is not.

1. The progress bar is an indicator of _when it is safe to conclude there was no effect_. If it is reaching 100% progress, and still gray|grey, it is likely that there is no lift in your primary metric of at least the configured MDE size. Other metrics might show significant results, and might warrant shipping the treatment even though the primary metric is flat, however.
2. If the progress bar is _blue_, it means that your primary metric is better in the treatment variant, and that the difference is statistically significant. Because of the sequential analysis methods we use, _it is safe to stop the experiment and ship it, even if the progress bar is not close to full_.